AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java

```diff
@@ -1954,7 +1971,9 @@ protected void displayCardAnswer() {
             long delay = mWaitQuestionSecond * 1000 + mUseTimerDynamicMS;
             if (delay > 0) {
                 mTimeoutHandler.removeCallbacks(mShowQuestionTask);
-                mTimeoutHandler.postDelayed(mShowQuestionTask, delay);
+                if(!mSpeakText){
```

Comment:
Same as above - whitespace changes only to match project standards

```suggestion
                if (!mSpeakText) {
```